src/query/cellxgene_census/config.vsh.yaml

```diff
@@ -77,14 +277,13 @@ functionality:
       path: test.py
 platforms:
   - type: docker
-  # TODO: increase python version
-    image: python:3.9
+    image: python:3.11
     setup:
       - type: python
         __merge__: [/src/base/requirements/anndata_mudata.yaml, .]
         packages:
-          - cellxgene-census~=1.2.0
-          - obonet~=1.0.0
+          - cellxgene-census
+          - scanpy
```

Comment:
Could you use `base/requirements/scanpy.yaml` with a `__merge__`?

Reply:
+1